### DISCUSSION TITLE
Hotkeys not working anymore

### DISCUSSION THREAD

**Discussion by ForeheadFox** [Belphemur/SoundSwitch#1174] [Q&A]
Heya, I used to use CTRL + F1 and CTRL + F3 as hotkeys.
CTRL + F1 was used for just switching between the recording devices.
CTRL + F3 was used for a profile I had made. 
Both of these hotkeys no longer work and I cannot change the binding to those two.
However my CTRL + F2 still works for playback devices.
I tried to reinstall and made sure to delete all the settings but I am still facing the same issue.
Any ideas?

**Comment by Cuffuf** [comment]
Same issue-- did you ever figure it out?

**Comment by ForeheadFox** [comment]
Its been a while but I remember having to remap to something different.On 18 May 2024 16:21, Cuffuf ***@***.***> wrote:
Same issue-- did you ever figure it out?
